ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -103,9 +110,9 @@ open class CommandBar: UIView {
         scrollView.translatesAutoresizingMaskIntoConstraints = false
         scrollView.contentInset = UIEdgeInsets(
             top: 0,
-            left: leadingButton == nil ? CommandBar.insets.left : CommandBar.fixedButtonSpacing,
+            left: leadingButton == nil ? CommandBar.insets.left : commandBarTokens.itemInterspace,
```

Comment:
nit: store "commandBarTokens.itemIterspace" in a constant (let) and use it in both of these lines (113 and 115).

Reply:
Same for lines 179, 181, 193 and 195.